### PR TITLE
ob support

### PR DESCRIPTION
**Pull request by hhxsv5**
Implements:
```PHP
Route::get('/test', function () {
    echo 'Test-', date('Y-m-d H:i:s');
});
```

### CODE REVIEW

File: src/Base.php [hunk 1]
```diff
@@ -84,6 +87,15 @@ public function onRequest($request, $response, $illuminate_request = false)
             $this->app['events']->fire('laravoole.on_request', [$illuminate_request]);
 
             $illuminate_response = $kernel->handle($illuminate_request);
+
+            $content = $illuminate_response->getContent();
+
+            if (strlen($content) === 0) {
+                $obContent = ob_get_contents();
+                strlen($obContent) !== 0 AND $illuminate_response->setContent($obContent);
```
Comment (denghongcai) [inline src/Base.php]:
```
ob_get_length() > 0 AND $illuminate_response->setContent(ob_get_contents());
```

Comment (hhxsv5) [reply src/Base.php]: Yeah

**Review by hhxsv5 — COMMENTED** [review]
Yeah

**Review by hhxsv5 — COMMENTED** [review]
check ob length

**Comment by hhxsv5** [comment]
还有什么问题，可以合了吗？

**Comment by garveen** [comment]
@denghongcai I have no more comments, you can merge it

**Comment by hhxsv5** [comment]
看发个release么？

**Comment by denghongcai** [comment]
+ garveen/laravoole@0.2.6

**Pull request merged**